rclrs_tests/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model
+   href="http://download.ros.org/schema/package_format3.xsd"
+   schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rclrs_tests</name>
+  <version>0.1.0</version>
```

Comment:
Please change this to 0.2.0, when we eventually submit these packages to the ROS buildfarm, they all must have the same version. Thanks.

Reply:
This one is not yet done.

Reply:
Ahh, I changed the xml version instead, sorry!

Reply:
Thank you! Could you also change the version number in `Cargo.toml` to 0.2?